test/extended/networking/util.go

```diff
@@ -157,14 +183,14 @@ func pluginIsolatesNamespaces() bool {
 		return true
 	}
 	// Assume that only the OpenShift SDN "multitenant" plugin isolates by default
-	return networkPluginName() == networkutils.MultiTenantPluginName
+	return openshiftSDNMode() == networkutils.MultiTenantPluginName
 }
 
 func pluginImplementsNetworkPolicy() bool {
 	switch {
 	case os.Getenv("NETWORKING_E2E_NETWORKPOLICY") == "true":
 		return true
-	case networkPluginName() == networkutils.NetworkPolicyPluginName:
+	case networkPluginName() == openshiftSDNPluginName || networkPluginName() == OVNKubernetesPluginName:
```

Comment:
you also need to check that the openshift-sdn mode is set to policy.

Reply:
Ack got it